packages/esm-commons-app/translations/en.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Not sure why, but I had to manually add these translations in, even though there is an `extract-translations` pre-commit hook. Projects like `openmrs-esm-patient-management` automatically updates this file on commit.

Reply:
Would be good to see if we can get an answer to why that is.